ScaleformUI_Lua/src/sidepanels/UIVehicleColorPickerPanel.lua

```diff
@@ -13,7 +14,7 @@ function UIVehicleColorPickerPanel.New(side, title, color)
         PanelSide = side,
         Title = title,
         TitleColor = _titleColor,
-        TitleType = _titleType,
+        TitleType = nil,
```

Comment:
titleType can be 0 or 1 or 2  can't be null... add it as last parameter and check if it's null